parallel-install/pkg/jobmanager/loggingJobs.go

```diff
@@ -0,0 +1,93 @@
+package jobmanager
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
+	istio "istio.io/client-go/pkg/clientset/versioned"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Register job using implemented interface
+
+type increaseLoggingPvcSize struct{}
+
+// No deprecation planned; 06/2021
+var _ = register(increaseLoggingPvcSize{})
+
+func (j increaseLoggingPvcSize) when() (component, executionTime) {
+	return component("logging"), Pre
+}
+
+func (j increaseLoggingPvcSize) identify() jobName {
+	return jobName("increaseLoggingPvcSize")
+}
+
+// This job increases the PVC-size of the logging component to 30GB.
+// This will be triggered before the deployment of its corresponding component.
+func (j increaseLoggingPvcSize) execute(cfg *config.Config, kubeClient kubernetes.Interface, ic istio.Interface, ctx context.Context) error {
```

Comment:
cfg is not used can be removed

Reply:
But what if some jobs want to use it, and since the Job needs to implement the interface, it has to stay. But lets have a talk on Monday regards this